commit 2f8b6fe5e51aa592637a0d9c3ad817978d8a2157 (tag: v2.2.0, refs/changes/84/684/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-10-13T15:52:25-07:00 (3 years ago)
    
    joyent/node-fast#2 update dependencies to support Node v4
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 108d54a..8a4ba05 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,10 @@
 
 No changes.
 
+## v2.2.0
+
+* #2 update dependencies to support Node v4
+
 ## v2.1.0
 
 * #1 need limited support for servers that send null values
diff --git a/package.json b/package.json
index ca21e25..b675ef8 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "fast",
 	"description": "streaming JSON RPC over TCP",
-	"version": "2.1.0",
+	"version": "2.2.0",
 	"main": "./lib/fast.js",
 	"repository": {
 		"type": "git",
@@ -14,7 +14,7 @@
 		"crc": "0.3.0",
 		"extsprintf": "^1.3.0",
 		"jsprim": "^1.2.2",
-		"kang": "^1.0.1",
+		"kang": "^1.1.0",
 		"lstream": "0.0.4",
 		"microtime": "2.0.0",
 		"posix-getopt": "1.2.0",
